### PR TITLE
ART-10881 Store Brew bundle builds in BigQuery

### PR DESCRIPTION
**Pull request by locriandev**
Ref. [ART-10881](https://issues.redhat.com/browse/ART-10881)

Test build: https://art-jenkins.apps.prod-stable-spoke1-dc-iad2.itup.redhat.com/job/hack/job/dpaolell/job/olm-bundle/2/console

Needs to go along with https://github.com/openshift-eng/aos-cd-jobs/pull/4260

### CODE REVIEW

File: doozer/doozerlib/cli/olm_bundle.py [hunk 1]
```diff
@@ -201,42 +210,109 @@ def rebase_and_build(olm_bundle: OLMBundle):
...
+                'pipeline_commit': 'n/a'
+            }
+
+            if record['status'] == 0:
+                image_pullspec = build_info['extra']['image']['index']['pull'][0]
+
+                build_record_params.update({
+                    'version': build_info['version'],
+                    'release': build_info['release'],
+                    'el_target': f'el{isolate_el_version_in_release(build_info["release"])}',
```
Comment (thegreyd) [inline doozer/doozerlib/cli/olm_bundle.py:268]:
```suggestion
                    'el_target': f'el{isolate_el_version_in_release(build_info["version"])}',
```

**Review by vfreex — COMMENTED** [review]
Thanks for the PR. After you address the issue that Sid's pointed out, I am ok with merging this.

**Comment by locriandev** [comment]
/hold

It's being discussed if bundle builds should have their own table, as we might want to store operands NVRs and image labels (e.g., `com.redhat.delivery.prerelease`)

**Comment by locriandev** [comment]
Superseded by https://github.com/openshift-eng/art-tools/pull/995

**Pull request closed without merging**